redash/query_runner/mongodb.py

```diff
@@ -301,19 +302,20 @@ def run_query(self, query, user):
 
         cursor = None
         if q or (not q and not aggregate):
-            if s:
-                cursor = db[collection].find(q, f).sort(s)
+            if "count" in query_data:
```

Comment:
Seems like indentation/order changed here? Was it intentional?

Reply:
Yes, this is intentional. We don't need to run the `find` if it is a count query.